cmd/conformance/gcp/kmsnote/main.go

```diff
@@ -45,13 +45,6 @@ import (
 	"k8s.io/klog/v2"
 )
 
-const (
-	// KeyVersionNameFormat is the GCP resource identifier for a key version.
-	// google.cloud.kms.v1.CryptoKeyVersion.name
-	// https://cloud.google.com/kms/docs/reference/rest/v1/projects.locations.keyRings.cryptoKeys.cryptoKeyVersions
```

Comment:
This link looks useful. Maybe move this to the `key_id` flag comment or the package doc?